src/reachy2_sdk/orbita/orbita2d.py

```diff
@@ -74,6 +75,10 @@ def __init__(
             "axis_1": getattr(self, axis1_name),
             "axis_2": getattr(self, axis2_name),
         }
+        self._reversed_joints = {
```

Comment:
this is not super important but it could be made automatically: https://stackoverflow.com/questions/483666/reverse-invert-a-dictionary-mapping